src/components/Proposals/ProposalBanner.js

```diff
@@ -73,19 +72,40 @@ function ProposalBanner({ onCloseBanner, ...props }) {
             color: ${theme.contentSecondary};
           `}
         >
-          We believe humankind should use technology as a liberating tool to
-          unleash all the goodwill and creativity of our species, rather than as
-          a tool to enslave and take advantage of one another.
+          This is a test <br /> Aragon Network Cash is a test organization set
```

Comment:
Is this the final campaign copy? it reads a little strange to me in this configuration. Should "This is a test" be in the paragraph? or is it intended as the title?